woocommerce/woocommerce-android
# [Google for Woo] Dashboard Card UI

Part of: #11963


### Description


This PR adds the "Google Ads Campaigns" card, as can be seen on My Store and on its Customize screen.

After that, it also adds the UI for Google Ads dashboard card, with designs for the following states:
- Card is loading
- Card when there's no campaigns found
- Card when it has a campaign

There is no functionality added yet.

### Testing information


Initially I want to implement something like the design on the iOS side:

| No campaigns found | Has campaigns |
|-|-|
| ![Simulator Screenshot - iPhone 15 Pro - 2024-07-11 at 16 25 39](https://github.com/user-attachments/assets/fb9d242a-f0e3-4808-979f-cbd9b645f2d7) | ![Simulator Screenshot - iPhone 15 Pro - 2024-07-11 at 16 25 19](https://github.com/user-attachments/assets/6708e136-6bc9-4718-b4d3-401a5fbb9c5a) |

This is what's added in this PR (please only check the first card on each screenshot):

| Loading | No campaigns found | Has campaigns |
|-|-|-|
| ![loading](https://github.com/user-attachments/assets/71626e68-c83a-43d9-938a-97358fd28c78) | ![no-campaigns](https://github.com/user-attachments/assets/b98abbf4-9294-45d3-a343-7035e1a4522b) | ![has-campaigns](https://github.com/user-attachments/assets/ac16693c-1327-4e82-90fc-c49ab2291151) |


To test:
1. Run the app, doesn't matter which store to use since for this PR I made it available on all sites,
2. In My Store, edit the card and enable "Google Ads Campaigns",
3. Go back to My Store and ensure all three states (loading, no campaigns, and has campaigns respectively) are shown. Check the design.
